frontend/simple/js/state.js

```diff
@@ -174,7 +196,8 @@ const actions = {
   async logout (
     {dispatch, commit, state}: {dispatch: Function, commit: Function, state: Object}
   ) {
-    await dispatch('saveSettings')
+    debouncedSave.cancel()
+    await dispatch('saveSettings', state)
     await db.clearCurrentUser()
     for (let key of Object.keys(state.contracts)) {
       await backend.unsubscribe(key)
```

Comment:
I don't remember where the issue for this (did we forget to open it??) but the backend should have a copy of the "rooms" and automatically keep it in sync with state.contracts, so once that's implemented this should be deleted.